app/views/hearings/show.html.haml

```diff
@@ -0,0 +1,75 @@
+= govuk_page_title(@hearing_day.strftime('%d/%m/%Y'), t('generic.hearing'))
+
+%table.govuk-table
+  %caption.govuk-table__caption.govuk-visually-hidden
+    = 'Hearing'
+  %tbody.govuk-table__body
+    %tr.govuk-table__row
+      %td.govuk-table__cell{ class: 'govuk-!-font-weight-bold' }
+        Hearing type
+      %td.govuk-table__cell
+        = @hearing.hearing_type
+    %tr.govuk-table__row
+      %td.govuk-table__cell{ class: 'govuk-!-font-weight-bold' }
+        Court
+      %td.govuk-table__cell
+        = @hearing.court_name || 'not set'
```

Comment:
Could we reduce / remove conditional logic from views?

@jsugarman

Reply:
@naseberry this is all still a bit up in the air so would prefer to leave for now. Eventually it would be good to get a decorator pattern in to deal with this logic

Reply:
that's ok then :+1: